vue.config.js

```diff
@@ -33,9 +33,14 @@ module.exports = {
   },
   devServer: {
     proxy: {
-      '^/userprofile': {
+      '(^/userprofile|^/graphql)': {
         target: 'http://localhost:3000/',
         changeOrigin: true
+      },
+      '^/subscriptions': {
+        target: 'http://localhost:3000/',
+        changeOrigin: true,
+        ws: true
```

Comment:
This URL needs a new block for the `ws` flag of the `webpack-dev-server` - https://github.com/webpack/webpack-dev-server/issues/283#issuecomment-162163329